
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following TypeErrorImpl was thrown building LayoutBuilder:
Unexpected null value.

The relevant error-causing widget was:
  LayoutBuilder
  LayoutBuilder:file:///C:/devs/django/bursary_home/flutter_apps/bursary_home/bursary_home_ui/lib/widgets/auth_layout.dart:19:12

When the exception was thrown, this was the stack:
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 307:3     throw_
errors.dart:307
dart-sdk/lib/_internal/js_dev_runtime/private/profile.dart 117:39               nullCheck
profile.dart:117
packages/bursary_home_ui/widgets/auth_layout.dart 44:32                         <fn>
auth_layout.dart:44
packages/flutter/src/widgets/layout_builder.dart 201:70                         updateChildCallback
layout_builder.dart:201
packages/flutter/src/widgets/framework.dart 3056:11                             buildScope
framework.dart:3056
packages/flutter/src/widgets/layout_builder.dart 240:5                          [_rebuildWithConstraints]
layout_builder.dart:240
packages/flutter/src/rendering/object.dart 2827:9                               <fn>
object.dart:2827
packages/flutter/src/rendering/object.dart 1161:7                               [_enableMutationsToDirtySubtrees]
object.dart:1161
packages/flutter/src/rendering/object.dart 2826:7                               invokeLayoutCallback
object.dart:2826
package